src/ontology/uberon-edit.obo

```diff
@@ -224080,41 +224046,29 @@ name: existence ends during
 def: "Relation between continuant c and occurrent s, such that every instance of c ceases to exist during some s, if it does not die prematurely." [https://orcid.org/0000-0002-6601-2165]
 synonym: "ceases_to_exist_during" EXACT []
 xref: RO:0002492
-is_a: existence_ends_during_or_before ! existence ends during or before
-transitive_over: part_of ! part of
 
 [Typedef]
 id: existence_ends_during_or_before
 name: existence ends during or before
 xref: RO:0002497
-holds_over_chain: developmentally_succeeded_by existence_ends_during_or_before
-holds_over_chain: part_of existence_ends_during_or_before
-transitive_over: part_of ! part of
-transitive_over: precedes ! precedes
-transitive_over: simultaneous_with ! simultaneous_with
 
 [Typedef]
 id: existence_ends_with
 name: existence ends with
 def: "Relation between continuant and occurrent, such that c ceases to exist at the end of p." [https://orcid.org/0000-0002-6601-2165]
 xref: RO:0002493
-is_a: existence_ends_during ! existence ends during
 
 [Typedef]
 id: existence_starts_and_ends_during
 name: existence starts and ends during
 xref: RO:0002491
-is_a: existence_ends_during ! existence ends during
-is_a: existence_starts_during ! existence starts during
 
 [Typedef]
 id: existence_starts_during
 name: existence starts during
 def: "Relation between continuant c and occurrent s, such that every instance of c comes into existing during some s." [https://orcid.org/0000-0002-6601-2165]
```

Comment:
Why not delete?

Reply:
The definitions are quite different. 
From `ro_import.owl` we have "x existence ends during y if and only if the time point at which x ends is before or equivalent to the time point at which y ends and after or equivalent to the point at which y starts. Formally: x existence ends during y iff ω(x) &lt;= ω(y) and ω(x) &gt;= α(y)."

Reply:
Ask in OBO uberon slack, stating that the same relation "x" has two definitions, and whether it is ok to delete the uberon one.

Reply:
I'm listing all the different definitions, and I'll ask there.